src/App.tsx

```diff
@@ -1,156 +1,256 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import classNames from 'classnames';
 import { UserWarning } from './UserWarning';
 
-const USER_ID = 0;
+import { Todo } from './types/Todo';
+import {
+  getTodos,
+  addTodos,
+  changeTodo,
+  deleteTodos,
+} from './api/todos';
+import { TodoList } from './components/TodoList/TodoList';
+import { TodoFilter } from './components/TodoFilter/TodoFilter';
+
+const USER_ID = 6861;
+
+enum Error {
+  add = 'add',
+  update = 'update',
+  delete = 'delete',
+}
 
 export const App: React.FC = () => {
+  const [todosOriginal, setTodosOriginal] = useState<Todo[] | undefined>();
+  const [todos, setTodos] = useState<Todo[] | undefined>();
+  const [activeTodo, setActiveTodo] = useState(0);
+  const [statusTodo, setStatus] = useState('All');
+  const [all, setAll] = useState(true);
+  const [active, setActive] = useState(false);
+  const [completedTodo, setCompletedTodo] = useState(false);
+  const [query, setQuery] = useState('');
+  const [error, setError] = useState<Error | null>(null);
+  const [idTodo, setIdTodo] = useState(0);
+
   if (!USER_ID) {
     return <UserWarning />;
   }
 
+  const fetchTodos = async () => {
+    const todosArr: Todo[] = await getTodos(USER_ID);
+
+    const activeCount = todosArr.filter(el => !el.completed).length;
```

Comment:
what if will be some server error? (you don't handle it here nor in useEffect, f.e)